cobalt/network/switches.cc

```diff
@@ -55,6 +55,15 @@ const char kMaxNetworkDelayHelp[] =
 const char kDisableInAppDial[] = "disable_in_app_dial";
 const char kDisableInAppDialHelp[] = "Disable the in-app dial server.";
 
+const char kQuicConnectionOptions[] = "quic_connection_options";
+const char kQuicConnectionOptionsHelp[] =
```

Comment:
Where can I find information about the available options for this? Can we point to that? A quick google search gave few results

Reply:
Maybe we can just point people to `net/third_party/quiche/src/quiche/quic/core/crypto/crypto_protocol.h`? Or if there's a more informative file, even better

Reply:
They are tags in crypto_protocol.h, but the available tags vary by QUICHE version, and whether they are connection_options or client_connection_options varies by tag and is not documented. I think it's best to leave it not documented because the documentation then has a large chance to be wrong or outdated.

Reply:
Well we don't have to worry about the version issue because we can point people to the actual header file we use—I think pointing people to that file in our codebase and saying something vague about it containing the tags we use would at least give people a jumping off point. I had no idea what AKDU meant before looking at that file or what other tags I could even try